src/main/resources/com/xpandit/plugins/xrayjenkins/task/XrayImportBuilder/config.jelly

```diff
@@ -156,7 +157,43 @@
 
             }
 
-            switcherLastConfiguration_${descriptor.defaultBuildID()} = function(){
+            switchTestInfoInput_${descriptor.defaultBuildID()} = function (container) {
+            	//current selected input type, from the select HTML element
+            	var selectTestInfoInputType = Q("#selectTestInfoInputType_${descriptor.defaultBuildID()}").find('option:selected').val();
+
+            	if (!selectTestInfoInputType)
+            		return;
+
+            	var key = Q("#selectFormat_${descriptor.defaultBuildID()}").find('option:selected').val();
+
+            	//selected input type, from last configuration
+            	var lastSwitcherConfig = '${instance.getInputTestInfoSwitcher()}';
+
+            	//field to render
+            	var _field = (formats[key]).configurableFields.filter(function (field) {
+            		return field.key == 'testImportInfo'
+            	})[0];
```

Comment:
You are assuming that thus field is always found, if that doesn't happen, you will get an error

Reply:
This field should always be present even if its value is empty.